app-modules/audit/src/Filament/Resources/AuditResource/Pages/ListAudits.php

```diff
@@ -74,6 +75,9 @@ public function table(Table $table): Table
                 TextColumn::make('event')
                     ->label('Event')
                     ->sortable(),
+                TextColumn::make('created_at')
+                    ->label('Occurred At')
+                    ->formatStateUsing(fn (string $state) => Carbon::parse($state)->format('d-m-Y h:i A')),
```

Comment:
Please make sure to use the m-d-Y formatting present in all of our apps.

```suggestion
                    ->formatStateUsing(fn (string $state) => Carbon::parse($state)->format('m-d-Y h:i A')),
```